README.md

```diff
@@ -70,7 +70,7 @@ $ pip install virtualenv
 ### Create virtualenv
 
 ```bash
-$ virtualenv --no-site-packages <myvirtualenvironmentname>
+$ virtualenv <myvirtualenvironmentname>
 ```
 
 - use no site packages to prevent virtualenv from seeing your global packages
```

Comment:
remove line 76: `- use no site packages to prevent virtualenv from seeing your global packages`

Reply:
ok done